README.md

```diff
@@ -1,15 +1,17 @@
 ASP.NET Core MVC
 ===
 
-**Note: For ASP.NET MVC 5.x, Web API 2.x, and Web Pages 3.x (not ASP.NET Core), please go to https://github.com/aspnet/AspNetWebStack**
+**Note: For ASP.NET MVC 5.x, Web API 2.x, and Web Pages 3.x (not ASP.NET Core), see https://github.com/aspnet/AspNetWebStack**
 
 AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/969jbosi0qwc1awg/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/mvc/branch/dev)
 
 Travis:   [![Travis](https://travis-ci.org/aspnet/Mvc.svg?branch=dev)](https://travis-ci.org/aspnet/Mvc)
 
 ASP.NET Core MVC gives you a powerful, patterns-based way to build dynamic websites that enables a clean separation of concerns and gives you full control over markup for enjoyable, agile development. ASP.NET Core MVC includes many features that enable fast, TDD-friendly development for creating sophisticated applications that use the latest web standards.
 
-ASP.NET Core MVC in ASP.NET Core includes support for building web pages and HTTP services in a single aligned framework that can be hosted in IIS or self-hosted in your own process.
+ASP.NET Core MVC includes support for building web pages and HTTP services in a single aligned framework that can be hosted in IIS or self-hosted in your own process.
+
+The [ASP.NET Core documentation](https://docs.microsoft.com/aspnet/core/).
```

Comment:
Per MS style guide, use complete sentences in documentation. Just kidding, no idea if it says that 😄 

And also, would it make sense to link to the MVC docs specifically, instead of the general ASP.NET Core docs? https://docs.microsoft.com/en-us/aspnet/core/mvc/overview